src/Io/Sender.php

```diff
@@ -72,9 +72,12 @@ public function __construct(HttpClient $http, MessageFactory $messageFactory)
      *
      * @internal
      * @param RequestInterface $request
+     * @param array $options Associative array containing the following options:
+     *                       'decodeContent' => [bool]: whether response body
+     *                       contents should be decoded (decompressed)
      * @return PromiseInterface Promise<ResponseInterface, Exception>
      */
-    public function send(RequestInterface $request)
+    public function send(RequestInterface $request, array $options = array())
```

Comment:
Not a big fan of this additional parameter. Any motivation for not keeping this in `withOptions()` exclusively?

Reply:
The motivation was simply that that class does not natively provide a withOptions mechanism. Should I add one?